bioimageio_collection_backoffice/_settings.py

```diff
@@ -15,9 +15,14 @@ class Settings(BaseSettings, extra="ignore"):
     )
 
     github_output: Optional[str] = None
+    id_parts: str = (
+        "https://raw.githubusercontent.com/bioimage-io/collection/main/id_parts.json"
+    )
+    """describes a file holding all parts to create bioimage.io resource ids"""
     reviewers: str = (
         "https://raw.githubusercontent.com/bioimage-io/collection/main/reviewers.json"
     )
+    """determines who is a bioimage.io reviewer"""
```

Comment:
Is this a comment, why not start with #?

Reply:
they are also mentioned as "attribute docstrings" in https://peps.python.org/pep-0257/#what-is-a-docstring